packages/react-tabs/src/components/Tab/useTabAnimatedIndicator.ts

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
```

Comment:
can we add tests for this hook ?

is there plan to add interaction/snapshot  test ?

Reply:
I have selection interaction tests, but not for the animation because it is so style intensive - maybe the screener tests would be a better place.  Haven't tested animations before.